spring-cloud-starter-dataflow-server-kubernetes/pom.xml

```diff
@@ -21,5 +21,51 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-server-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Even if we add the description, it looks like Boot grabs the first banner it finds, which seems to be the Data Flow core one - maybe we should move that one out of core and into the local starter